app/routes/oob.py

```diff
@@ -22,7 +22,35 @@ async def create_oob_invitation(
     auth: AcaPyAuth = Depends(acapy_auth_from_header),
 ) -> InvitationRecord:
     """
-    Create connection invitation out-of-band.
+    Create an out-of-band invitation.
+    ------------------------------
+
+    The attachment field is used to include a credential offer or a proof request in the invitation.
+    The attachment ID is the credential exchange ID or proof request ID.
+    The attachment type is either ```"credential-offer"``` or ```"present-proof"```.
+
+    The multi_use field is used to determine if the invitation can be used multiple times by different tenants.
+    The ```use_public_did``` field should only be set true, if a tenant with a public DID is creating
+    a connection invitation, then the invitation will use the tenants public did to create the connection invitation
+    i.e. the tenant accepting the invitation will connect via public did of tenant that created invitation
+
+    ```multi_use``` can't be set to `true` if an ```attachment``` is provided,
+    as a proof request or credential offer should be sent to one tenant.
+
+    Request body:
+    -------------
+        body:CreateOobInvitation
+            alias: Optional[str]
+            multi_use: Optional[bool]
+            use_public_did: Optional[bool]
```

Comment:
I think we should replace all cases of "Optional[bool]" with just bool and a default value, so it's still non-required but the default behaviour is more clear.

Optional bools are always a bit weird, because it's essentially adding a 3rd option to a true / false, and that only makes sense if it has some behavior that's distinct from just plain true / false